test/unit/org/apache/cassandra/hints/HintsStoreTest.java

```diff
@@ -137,6 +138,26 @@ public void testConcurrentDeleteExpiredHints() throws Exception {
         assertFalse("All hints files should be deleted", store.hasFiles());
     }
 
+    @Test
+    public void testPendingHintsInfo() throws Exception
+    {
+        HintsStore store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 0, Long.MAX_VALUE, Long.MIN_VALUE),
+                     store.getPendingHintsInfo());
+
+        final long t1 = System.currentTimeMillis();
```

Comment:
Please, use `Clock`, see https://issues.apache.org/jira/browse/CASSANDRA-16923 for details.

Reply:
Thanks for pointing it out, I was not aware of this issue! As the PR's base does not contain CASSANDRA-16923, I'll switch after all other issues are resolved before squashing the changes.

Reply:
Fixed in https://github.com/apache/cassandra/blob/db1959a7400008f56466945df8d8c24033df1d66/test/unit/org/apache/cassandra/hints/HintsStoreTest.java#L147-L148